src/components/FeeLabelToggle/FeeLabelToggle.stories.js

```diff
@@ -25,11 +25,32 @@ import FeeLabelToggle from './FeeLabelToggle';
 import WithThemeDecorator from '../../../storybook/WithThemeDecorator';
 import CenterViewDecorator from '../../../storybook/CenterViewDecorator';
 
+const reduxData = {
+  rates: {
+    BTC: {
+      ETH: 27.228736, EUR: 53175, GBP: 46132, USD: 63591,
+    },
+    ETH: {
+      ETH: 1, EUR: 2017.2, GBP: 1756.24, USD: 2407.1,
+    },
+    PLR: {
+      ETH: 0.00002224, EUR: 0.04489, GBP: 0.03906, USD: 0.05355,
+    },
+    WBTC: {
+      ETH: 27.233989, EUR: 53190, GBP: 46151, USD: 63604,
+    },
+    GBP: {
+      ETH: 27.233989, EUR: 53190, GBP: 46151, USD: 63604,
+    },
+  },
+};
+
 storiesOf('FeeLabelToggle', module)
   .addDecorator(CenterViewDecorator)
   .addDecorator(WithThemeDecorator)
   .add('default', () => (
     <FeeLabelToggle
```

Comment:
This `reduxData` is not passed correctly to the `FeeLabelToggle` as you use the FeeLabelToggle that is exported with `react-redux` `connect` HOC, which causes `connect` HOC to overwrite passed `reduxData` props.

In order to pass redux data this way, please add named export in FeeLabelToggle and import that one only for tests. 

You will then need to supply dummy data for remaining redux fields expected by FeeLabelToggle like this:
```
const reduxData = {
  accountAssets: {},
  accountHistory: [],
  baseFiatCurrency: defaultFiatCurrency,
  isGasTokenSupported: true,
  rates: {
    ETH: {
      ETH: 1,
      EUR: 2017.2,
      GBP: 1756.24,
      USD: 2407.1,
    },
  },
};
```